README_js.md

```diff
@@ -333,6 +333,7 @@ API is not supported. This issue can be resolved by adding an appropriate polyfi
 | Environment  | Suggested Polyfill                                                                                  |
 | ------------ | --------------------------------------------------------------------------------------------------- |
 | React Native | [`react-native-get-random-values`](https://github.com/LinusU/react-native-get-random-values#readme) |
+| WebWorker (Edge) | N/A (let us know if you find something) |
```

Comment:
Worth to mention only old edge have this problem

Reply:
Sure... do we have data on which Edge versions do/don't support crypto in webworkers?  The only thing I've found is the [caniuse page](https://caniuse.com/#feat=cryptography), which doesn't go into detail on versions.

Reply:
It does. 18 and below is old edge. 79+ is chromium based
![image](https://user-images.githubusercontent.com/5635476/75797363-1bb3f200-5d86-11ea-9e82-3679766c4c0e.png)